custom_typings/dom.d.ts

```diff
@@ -0,0 +1,11 @@
+declare interface Window {
+  customElements: CustomElementRegistry;
```

Comment:
That looks reasonable to me. I think we'd be willing to take that. Any objections @mhegazy? Would this go in https://github.com/Microsoft/TSJS-lib-generator @zhengbli?